extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -160,7 +167,8 @@ public void dispose() {
         if (lightId != null) {
             HueBridgeHandler bridgeHandler = getHueBridgeHandler();
             if (bridgeHandler != null) {
-                getHueBridgeHandler().unregisterLightStatusListener(this);
+                this.bridgeHandler.unregisterLightStatusListener(this);
+                this.bridgeHandler = null;
```

Comment:
But for the call to unregister I could still use the local reference like before.
In fact I could just add one line and keep the old one unchanged.
Will change that.